### PR TITLE
Improve gas estimation logs and signed data verification

### LINKED ISSUE #234: Prevent double error log during gas estimation
Consider changing https://github.com/api3dao/airseeker-v2/blob/e0f369fa433e0f9857a3d1643a57c7c39aa489f4/src/update-feeds-loops/submit-transactions.ts#L173 to INFO.

### LINKED ISSUE #236: Change when Airseeker warns about outdated signed data
See https://api3workspace.slack.com/archives/C05TQPT7PNJ/p1712218464326829?thread_ts=1712215557.278709&cid=C05TQPT7PNJ

### PR DESCRIPTION
**Pull request by Siegrift**
Closes https://github.com/api3dao/airseeker-v2/issues/234
Closes https://github.com/api3dao/airseeker-v2/issues/236

### CODE REVIEW

**Comment by bdrhn9** [comment]
Could I take a look after [that](https://github.com/api3dao/airseeker-v2/pull/233) PR is merged? I'm getting confused because it seems like changes are showing up for both PRs right now 😄

**Comment by Siegrift** [comment]
> Could I take a look after https://github.com/api3dao/airseeker-v2/pull/233 PR is merged? I'm getting confused because it seems like changes are showing up for both PRs right now 😄

No worries. I recommend looking commit by commit - it should make it a bit easier to review.

File: src/update-feeds-loops/submit-transactions.ts [hunk 1]
```diff
@@ -167,10 +168,21 @@ export const estimateMulticallGasLimit = async (
...
+  // It is possible that the gas estimation failed because of a contract revert due to timestamp check, because the feed
+  // was updated by other provider in the meantime. Try to detect this expected case and log INFO instead.
+  if (errorMessage.includes('Does not update timestamp')) {
+    logger.info(`Gas estimation failed because of a contract revert.`, { errorMessage });
+  } else {
+    logger.warn(`Unable to estimate gas for multicall using provider.`, { errorMessage });
+  }
 
   if (!fallbackGasLimit) {
-    throw new Error('No fallback gas limit provided.');
```
Comment (bdrhn9) [inline src/update-feeds-loops/submit-transactions.ts:173]: Looks like we don't need to utilize promise-utils while calling this function `estimateMulticallGasLimit` above.

**Review by bdrhn9 — APPROVED** [review]
LGTM 🔥

File: src/update-feeds-loops/submit-transactions.ts (hunk 1 above)
Comment (Siegrift) [reply src/update-feeds-loops/submit-transactions.ts:173]: Yeah, thx.

**Pull request merged**